### PR TITLE
Feature/frontend/enhancement cookie consent

### PR DESCRIPTION
**Pull request by Jasonwei1106**
This change is [Reviewable](https://reviewable.io/reviews/xcov19/mycovidconnect/103)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
         "node-gyp": "^7.1.0",
         "node-sass": "^4.0.0",
         "react": "^16.13.1",
+        "react-cookie-consent": "^5.1.3",
```
Comment (codecakes) [inline package.json:21]: why have you added this? see the user story

**Review by codecakes — CHANGES_REQUESTED** [review]
1. Take a fresh git pull and merge your branch from master first.
Resolve the conflicts.
2. Then look at MYC-73 and MYC-113

File: package.json (hunk 1 above)
Comment (Jasonwei1106) [reply package.json:21]: This is like jquery but in the react version. I used the library for React.

Comment (codecakes) [reply package.json:21]: there's no need for this. remove it and checkout the jira ticket. everything is there on what's needed. or sync up with @vasynizami

**Pull request closed without merging**